zng/typetype_test.go

```diff
@@ -0,0 +1,18 @@
+package zng_test
+
+import (
+	"testing"
+
+	"github.com/brimdata/zed/zng"
+	"github.com/brimdata/zed/zson"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTypeValue(t *testing.T) {
+	const s = "{A:{B:int64},C:int32}"
+	zctx := zson.NewContext()
+	typ, err := zson.ParseType(zson.NewContext(), s)
+	require.NoError(t, err)
+	tv := zctx.LookupTypeValue(typ)
```

Comment:
```suggestion
	typ, err := zson.ParseType(zson.NewContext(), s)
	require.NoError(t, err)
	tv := zson.NewContext().LookupTypeValue(typ)
```

Reply:
I had used that test to debug I a problem I had in early code.  It was ok to use the same context.  I will add your logic as a second test.